client/components/Auth/AuthAlert.vue

```diff
@@ -0,0 +1,28 @@
+<template>
```

Comment:
It seems strange to me that this component would take a list of errors. A more natural API would be to pass a string to this component with the error message. 

For example when I tried and failed to login I got a funny set of error messages:
<img width="552" alt="image" src="https://user-images.githubusercontent.com/45678264/181876904-354be6e0-aaf9-4765-a547-ac4cd964453e.png">


I would have expected the parent to pass down a nicer string like:
```
<AuthAlert error=":errorMessage" />
```

Where `errorMessage` is a computed property based on the content of the errors we have. So we could have a message like 'User doesn't exist', or whatever

Reply:
I kept it a list of errors because that's what it is on main.
I suspect it's to handle multiple errors being raised by Axios? (If that's even possible)

Reply:
Django Rest Framework outputs an array of error messages for validation errors. Though I suppose we would never actually have multiple error messages for an error, but I had the v-for directive just in case.

Reply:
> I kept it a list of errors because that's what it is on main.
Fair enough, maybe a TODO to think about how we can render nicer messages though. No user is going to understand a 500 error is. We can keep the list API if we don't want to make a breaking change, but we should think about what those messages actually say

Reply:
I haven't had a chance to see how error alerts are implemented in this branch, but on main it displays DRF validation errors 
![Screenshot_20220801_215212](https://user-images.githubusercontent.com/40915904/182163470-e48e36a6-2828-400e-81f6-02de66522413.png)

Reply:
Yeh ok, we can refine it later, this is fine for now I think

Reply:
> I haven't had a chance to see how error alerts are implemented in this branch, but on main it displays DRF validation errors (for the signup page) ![Screenshot_20220801_215212](https://user-images.githubusercontent.com/40915904/182163470-e48e36a6-2828-400e-81f6-02de66522413.png)


It's the same on this branch.